CHANGES

```diff
@@ -1,6 +1,11 @@
 Version 8.16 (Unreleased)
 -------------------------
 - Added data migration to create UserEmail objects for users whose primary emails did not have them
+- Added distributions to the release system to better support mobile apps
+
+Version 8.15
+------------
+
 - Time series data (used by graphs and other features) is now updated when groups are merged.
```

Comment:
We got this pattern that we've used in every version: https://github.com/getsentry/sentry/blob/master/CHANGES#L32-L45